lib/jekyll/utils/platforms.rb

```diff
@@ -13,18 +13,40 @@ module Platforms
         end
       end
 
+      #
+
+      def windows?
+        !!(RbConfig::CONFIG["host_os"] =~ %r!mswin|mingw|cygwin!i || \
+          proc_version =~ %r!microsoft!i)
+      end
+
+      #
+
+      def linux?
+        RbConfig::CONFIG["host_host"] =~ %r!linux! && \
+          !proc_version =~ %r!microsoft!i
```

Comment:
You're right, I was in a hurry and doing it on my tablet, I'll ship a correction.